app/models/organization.rb

```diff
@@ -3,4 +3,8 @@ class Organization < ActiveRecord::Base
   validates :department_name, presence: true
   validates :team_name, presence: true
   validates :agency_name, uniqueness: { scope: [:department_name, :team_name] }
+
+  def to_s
+    "#{agency_name} | #{department_name} | #{team_name}"
```

Comment:
I am not 100% sure should have both `department` and `agency` -- maybe it should be an OR? 

see https://en.wikipedia.org/wiki/Government_agency#United_States